lib/ar2dto/converter.rb

```diff
@@ -24,17 +24,21 @@ def apply_configs(options)
 
     def add_methods(hash)
       options&.dig(:methods)&.each do |method|
-        result = model.send(method)
-        result = if result.respond_to?(:to_dto)
-                   result.to_dto
-                 else
-                   result.as_json
-                 end
-        hash[method.to_s] = result
+        hash[method.to_s] = data_only(model.send(method))
       end
       hash
     end
 
+    def data_only(object)
+      if object.respond_to?(:to_dto)
+        object.to_dto
```

Comment:
I know it was already present, but I wonder if by calling this `.to_dto` it will automatically apply all the options?

Reply:
global and model options will be respected, but we can't pass option to to_dto because we wouldn't know what to pass there. If the method needs something different it can call `.to_dto(options)` before returning it, I guess. WDYT?